test/gurka/test_phonemes.cc

```diff
@@ -379,7 +381,7 @@ TEST(Standalone, PhonemesWithAltandDirection) {
     uint32_t sign_index = 0;
     ASSERT_EQ(signs.size(), 1);
     ASSERT_EQ(pronunciations.size(), 1);
-    for (const auto& sign : signs) {
+    for ([[maybe_unused]] const auto& sign : signs) {
```

Comment:
Another possible solution is to use a for-loop
```
for (uint32_t sign_index = 0; sign_index < signs.size(); ++sign_index) {
```

Reply:
this is fine IMO

Reply:
size is one, I will delete the loop